lib/debug.py

```diff
@@ -252,7 +252,7 @@ def init_debugger(self):
                 if self.debugger == 'vscode':
                     debugpy.listen(('localhost', 5678))
                     if self.wait_attach:
-                        print("Waiting for debugger attach")
+                        self.log("Waiting for debugger attach")
```

Comment:
Would you mind not to change this file? There is an other pull request which will alter this file file completely. I think we could avoid unnecessary conflicts here.

Reply:
Reverted.